src/ansys/aedt/core/application/analysis.py

```diff
@@ -1269,13 +1094,17 @@ def get_nominal_variation(self, with_values=False):
 
         Returns
         -------
-        list of str
-            List of nominal variations.
+        dict
+
         """
+        independent_flag = self.available_variations.independent
+        self.available_variations.independent = True
         if not with_values:
-            return self.available_variations.nominal
+            variations = self.available_variations.nominal
```

Comment:
@Samuelopez-ansys it's a bit misleading because the docstring of this method says: "Retrieve the nominal variation." so it means a single variation but you are instantiating _variations_ (plural).
I guess this is done throughout the whole code in Analysis. Could you please check?